weasyl/character.py

```diff
@@ -234,15 +234,63 @@ def select_view(userid, charid, rating, ignore=True, anyway=None):
         "page_views": query[12],
         "friends_only": "f" in query[11],
         "hidden_submission": "h" in query[11],
-        # todo
-        "fave_count": define.execute("SELECT COUNT(*) FROM favorite WHERE (targetid, type) = (%i, 'f')",
-                                     [charid], ["element"]),
+        "fave_count": favorite.count(charid=charid),
         "comments": comment.select(userid, charid=charid),
         "sub_media": fake_media_items(charid, query[0], login, query[11]),
         "tags": searchtag.select(charid=charid),
     }
 
 
+def select_view_api(userid, charid, anyway=False, increment_views=False):
+    query = define.execute("""
```

Comment:
@charmander What's the best way to do this without positional results in query? Is it worth it?